dev-util/umockdev/umockdev-0.10.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+
+inherit python-any-r1
+
+MY_PV=${PV%??}
+
+DESCRIPTION="Mock hardware devices for creating unit tests"
+HOMEPAGE="https://github.com/martinpitt/umockdev/"
+SRC_URI="https://launchpad.net/${PN}/trunk/${PV}/+download/${PN}-${MY_PV}.tar.xz"
```

Comment:
Please substitute the first `${PN}` with fixed string here, it will make it easier for people to copy-paste into browser for lookup. You may also want to list launchpad as second homepage.